src/renderer/theme.store.ts

```diff
@@ -32,42 +34,46 @@ export enum ThemeType {
 }
 
 export interface Theme {
-  id: ThemeId; // filename without .json-extension
   type: ThemeType;
-  name?: string;
-  colors?: Record<string, string>;
-  description?: string;
-  author?: string;
+  name: string;
+  colors: Record<string, string>;
+  description: string;
+  author: string;
+}
+
+const darkTheme: Theme = {
+  ...darkThemeBase,
+  type: darkThemeBase.type as ThemeType,
+};
+
+const lightTheme: Theme = {
+  ...lightThemeBase,
+  type: lightThemeBase.type as ThemeType,
+};
```

Comment:
This is more verbose than it might otherwise seem necessary because `type` is TS typed as `string` instead of `ThemeType`. 

I could have done `lightThemeBase as Theme` but that wasn't throwing errors if I removed the `author` field from the JSON file. So this mechanism is more resilient to changes to the `interface Theme` type.